lib/datasource/remote/api/eos_repo/eos_repository.dart

```diff
@@ -74,7 +74,12 @@ abstract class EosRepository {
   }
 
   ErrorResult mapEosError(dynamic error) {
+    final regex = RegExp(r'^.*Internal Service Error.*assertion failure with message: ([^\"]*)');
```

Comment:
Not sure this is a valid assumption?! That all transaction errors come as "internal service error"??

Doesn't matter too much since it's just a log but...

Reply:
If it fits the regex then user sees the "clean" error message, which is better UX. If it doesn't match, then we see the whole string, so nothing lost.